src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/wrapper/ErrorHandlingWrapper.java

```diff
@@ -40,15 +41,16 @@ public ErrorHandlingWrapper(ErrorHandler errorHandler) {
         }
     }
 
-    public void wrapDocUploadFailure(Envelope envelope, Supplier<Void> supplier) {
+    public Boolean wrapDocUploadFailure(Envelope envelope, Supplier<Boolean> supplier) {
```

Comment:
now that this function returns a vault, a javadoc comment would be nice

Reply:
That's done.